platform/android/java/lib/src/org/godotengine/godot/input/GodotInputHandler.java

```diff
@@ -56,6 +56,9 @@
  */
 public class GodotInputHandler implements InputManager.InputDeviceListener {
 	private static final String TAG = GodotInputHandler.class.getSimpleName();
+	
+	private static final int VERTICAL_AXIS = 1;
+	private static final int HORIZONTAL_AXIS = 0;
```

Comment:
Let's update these constants to `ROTARY_INPUT_VERTICAL_AXIS` and `ROTARY_INPUT_HORIZONTAL_AXIS` for clarity.

Reply:
Fixed.